diff --git a/Modules/funct.f90 b/Modules/funct.f90
index 75758b5..156c883 100644
--- a/Modules/funct.f90
+++ b/Modules/funct.f90
@@ -908,7 +908,7 @@ CONTAINS
     !
     CHARACTER(LEN=256) :: name
     INTEGER :: i, l, prev_len(6), fkind, fkind_v(3), family
-    INTEGER, PARAMETER :: ID_MAX_LIBXC=600
+    INTEGER, PARAMETER :: ID_MAX_LIBXC=900
     TYPE(xc_f03_func_t) :: xc_func
     TYPE(xc_f03_func_info_t) :: xc_info
     LOGICAL, EXTERNAL :: matches
